sdk/include/xdk/winnt_old.h

```diff
@@ -2562,7 +2562,7 @@ typedef struct _RTL_CRITICAL_SECTION_DEBUG {
   DWORD ContentionCount;
   DWORD Flags;
   WORD CreatorBackTraceIndexHigh;
-  WORD SpareWORD;
+  WORD SpareUSHORT;
```

Comment:
This is called SpareWORD in PSDK, so it should stay unchanged.

Reply:
I saw in two places that this field is called in 6+, i made #ifdef here

Reply:
Those places are wrong. PSDK is the source of truth and it says SpareWORD.

Reply:
All three are wrong??? They get information from pdb throught WinDbg
https://www.nirsoft.net/kernel_struct/vista/RTL_CRITICAL_SECTION_DEBUG.html
http://winappdbg.sourceforge.net/doc/v1.5/reference/winappdbg.win32.peb_teb.RTL_CRITICAL_SECTION_DEBUG-class.html
http://terminus.rewolf.pl/terminus/structures/ntdll/_RTL_CRITICAL_SECTION_DEBUG_x86.html

You mean PSDK for 2003/XP

Reply:
What Windows calls this internally doesn't matter since these are public header files that are used to build applications.
Our compat target for these is the latest Windows SDK -- i.e. Win10 RS5.

Reply:
In my branch I already have this implemented. Wine uses it as debugger information storage, but that goes against latest Windows SDKs (I assume because it was implemented quite some time ago). But in my opinion the best way is this, compatible with both Wine and Windows SDK:
<https://github.com/andrew-boyarshin/reactos/blob/1b4aadfd93dce89ffe1598fc0a407c9b3f684b62/sdk/include/xdk/winnt_old.h#L2561-L2573>

Reply:
@ThFabba ok, rolled back to SpareWORD